src/overlays/actors/ovl_Boss_Sst/z_boss_sst.h

```diff
@@ -6,11 +6,58 @@
 
 struct BossSst;
 
+typedef void (*BossSstActionFunc)(struct BossSst*, GlobalContext*);
+
+typedef struct {
+    /* 0x0000 */ Vec3f pos;
+    /* 0x0010 */ Vec3f vel;
+    /* 0x0018 */ Vec3s rot;
+    /* 0x001E */ u16 scale;
+    /* 0x0020 */ s16 move;
+    /* 0x0022 */ s16 status;
+    /* 0x0024 */ u8 alpha;
+} BossSstEffect; // size = 0x28
+
+typedef struct {
+    /* 0x0000 */ PosRot posRot;
+    /* 0x0014 */ f32 zPosMod;
+    /* 0x0018 */ s16 yRotMod;
+} BossSstHandTrail; // size = 0x1C
+
 typedef struct BossSst {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x94C];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ BossSstActionFunc actionFunc;
+    /* 0x0194 */ union { s8 vanish; s8 parity;};
```

Comment:
A way to have that field have two different names to reflect the two different uses it has. Since we aren't doing that sort of thing, it needs to be changed, but I hadn't gotten around to it.

Reply:
I think this still needs to be addressed?

Reply:
Yeah it does. I'm still trying to think of a way to encompass both in one name.